src/utils/dig.js

```diff
@@ -28,23 +30,27 @@ export const validateDomain = (input, response) => {
     };
 };
 
-export const handleDig = async ({ domain, types, short, provider }) => {
+export const handleDig = async ({ domain, types, short, cdflag, provider }) => {
     // Make the DNS queries
     const results = await Promise.all(types.map(type =>
-        performLookupWithCache(domain, type, provider.doh).then(data => ({ type, data }))));
+        performLookupWithCache(domain, type, provider.doh, cdflag).then(data => ({ type, data }))));
+
+    const max_length = 4096 - (cdflag ? DNSSEC_DISABLED_WARNING_MESSAGE.length : 0);
 
     // Define the presenter
     const present = (type, data) => {
         // Generate the dig command equivalent
-        const digCmd = `\`${data.name} ${type} @${provider.dig} +noall +answer${short ? ' +short' : ''}\`\n`;
+        const digCmd = `\`${data.name} ${type} @${provider.dig} +noall +answer${short ? ' +short' : ''}${cdflag ? ' +cdflag' : ''}\`\n`;
 
         // Error message
         if (typeof data === 'object' && data.message)
             return `${digCmd}\n${data.message}`;
 
         // No results
         if (typeof data !== 'object' || !Array.isArray(data.answer) || data.answer.length === 0)
-            return `${digCmd}\nNo records found`;
+            return `${digCmd}\nNo records found${(cdflag
+                ? DNSSEC_DISABLED_WARNING_MESSAGE
+                : '')}`;
```

Comment:
```suggestion
            return `${digCmd}\nNo records found${cdflag
                ? `\n\n${DNSSEC_DISABLED_WARNING_MESSAGE}`
                : ''}`;
```
Double linebreak make sense for formatting here